REPORT/es_integration/flatten_to_es.py

```
@@ -314,8 +316,19 @@ def main():
     flattener = ESDocumentFlattener(index_name=args.es_index)
+    if args.output and Path(args.output).is_file():
+        try:
+            with open(args.output, 'r') as f:
+                for line in f:
+                    doc = json.loads(line)
+                    if 'batch_id' in doc:
+                        flattener.batch_id = doc['batch_id']
+                        print(f"Reusing existing batch_id: {flattener.batch_id}")
+                        break
+        except Exception:
+            pass
```

Ruff (0.16.0)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)